docs/Installation/InstallationOnClusters.md

```diff
@@ -20,9 +20,10 @@ want to load the modules, source the shell file for your system and run
    `SYSTEM_TO_RUN_ON` is replaced by the name of the system as described in the
    relevant section below.
 5. If you haven't already, choose where you want to install the dependencies,
-   e.g. into `SPECTRE_DEPS` and run `spectre_setup_modules SPECTRE_DEPS`. This
+   e.g. into `SPECTRE_DEPS` and run `spectre_setup_modules $SPECTRE_DEPS`. This
```

Comment:
It looks like this section was consistently using $ on SPECTRE_HOME and no $ on everything else.  If you are going to change this one please make the others consistent.

Reply:
make them all $SPECTRE_DEPS and add an export line and squash